### PR TITLE
Add 'My turn is for learning' pattern

### PR DESCRIPTION
**Pull request by alexanderbird**
This is something we did on the last team I was part of for helping new interns feel comfortable participating.

I haven't done a thorough review of what I wrote, but I thought it would be better to open a PR with it as-is than leave it on my hard drive for a month or two before I get to it again. (But if you're able to comment on it at all, that would probably motivate me to improve it. Otherwise if you wait long enough I'll probably come back and try to make it better on my own ;) )

You can preview the content as a Github page here: https://alexanderbird.github.io/Mobbing-Pattern-Language/patterns/My%20turn%20is%20for%20learning.html

Cheers!

### CODE REVIEW

**Comment by JayBazuzi** [comment]
Neat idea.

I usually instruct the mob to make sure the Navigator is never at a loss for what to do next. If they're unsure, they can ask the mob for ideas, and pick one. Maybe we need to document that, too.

It sounds like you've found a useful alternative for that situation, which is just what the pattern language is for!

**Comment by alexanderbird** [comment]
Yeah, there are many things to try before this one. I agree that providing the navigator with ideas (or having them lead by asking someone for advice) is preferable to this. In our case, the team member had a hard time processing the minutia without the big picture, so any they had a bit of a mental block with smaller goals. "Yeah I can add a method to process that input, but I have no idea what we are working towards." Without the big picture, they froze up -- and their instinct was to go work through things solo until they could wrap their head around it. 

As a related thing, they could understand mobbing for coding, but mobbing for learning, discovery, or documentation was a new thing.

So the idea I was trying to communicate here is "whatever you would do solo to move forward on this problem, you can do that with the mob. You don't have to just program." 

Maybe a corollary to this one is if the navigator before me is working through adding some new methods and we wrap that up, and I'm not sure of where we're going, when it's my turn to navigate I may have the mob create some block diagrams or do some other sketching to get a big picture. Then the next navigator may benefit from that diagram we made under my navigation to confidently move forward with the next method to implement, etc.

To sum up:
 - I do think there's something interesting to capture here, but I don't think I've succeeded
 - at the very least, this pattern should be more clear about the other things that are more effective and common than this

------
Maybe this pattern should be more generally "Learning navigator" and be about taking your turn to zoom out, do some analysis, answer some questions, etc.

For an experienced mob, that sort of thing can happen in the background -- but when a mob is less experienced or smaller it sometimes makes sense to make it a main, foreground activity.

And you don't need consensus to move to that sort of info gathering / discovery work. When you're navigating, if that seems wise or necessary to you, you can go for it.

Also the driver/navigator pattern is applicable for producing diagrams and documentation, conducting research, etc.
---

This pattern needs some refinement.

**Comment by JayBazuzi** [comment]
I'm OK with merging into master whenever you are ready. It doesn't have to be perfect - we can continue to improve content over time. It's up to you.

**Comment by alexanderbird** [comment]
Sure! I've added "Grease the wheels" as a placeholder for the other pattern you described -- feeding the navigator ideas. I wanted it more clear that that's a better first step, and the "My turn is for learning" is more of a fall-back plan. 

Now I'm comfortable merging it -- as you said it can be improved upon later.

File: _patterns/My turn is for learning.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+tags: introducing-mobbing
+---
+# Also Known As
+
+# Context
+
+One member of the mob is missing the context (techical, domain, or both) for the
```
[the remark this answers is not in the document]
Comment (alexanderbird) [reply _patterns/My turn is for learning.md]: "tyop" -- ironic.

Thanks for catching these, I didn't have spell check turned on in my editor 🤦

**Pull request merged**